charts/airflow/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Airflow Helm Chart (User Community) - used to deploy Apache Airflow on Kubernetes
 name: airflow
-version: 8.5.2
+version: 8.5.3
```

Comment:
Please do not change the version as this triggers a release.

Reply:
I changed it back to 8.5.2